spec/order_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'order'
+describe Order do
+    let(:hamburger) {double(name: 'Hamburger', price: 5)}
+    let(:chips) {double(name: 'Chips', price: 2)}
+
+    it {is_expected.to respond_to(:add, :total_price, :print_order, :place_order)}
+
+    it 'starts with empty order list' do
+        expect(subject.order_list).to be_empty
```

Comment:
You are testing behaviour vs state here. Have a think about how you want this class to be used, should you be able to reach in and look at the order from outside? If an attribute reader isn't used in your feature tests (eg. irb) then it shouldn't be part of the class, and therefore shouldn't be used in tests.